README.md

```diff
@@ -62,10 +53,10 @@ The [PureScript Book](https://leanpub.com/purescript/read) is the recommended ap
 
 - [Related Projects](Related-Projects.md)
 - [Differences from Haskell](language/Differences-from-Haskell.md)
-- [Alternate compiler backends](ecosystem/Alternate-backends.md) for various target languages
 
 ## TL;DR
-* [Install PureScript](http://www.purescript.org/download/)
-* [Install Node.js](https://nodejs.org/): `brew install node`/`pacman -S nodejs`/`choco install nodejs`/[Download Node.js](https://nodejs.org/)
-* [Install Pulp](https://github.com/bodil/pulp): `npm install -g pulp bower`
-* GO: `pulp init`, `bower install --save purescript-tuples`, `pulp build --to output.js`, etc.
+
+- [Install PureScript](http://www.purescript.org/download/)
```

Comment:
Maybe let's just remove everything under the TL;DR heading? The getting started guide didn't exist when this was written; now that it does I'm not sure this stuff should stay here. If people don't know this stuff already we should direct them to the getting started guide, because it's better maintained and easier to find.